### PR TITLE
Create header section

### PR DESCRIPTION
**Pull request by DuongPham-dev**
- #9

### CODE REVIEW

File: practices/practice-one/src/components/Header/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          <Heading tag="h1" title="Blog" />
+          <Paragraph content="Stay up to date with our portfolio" />
+        </section>
+        <section className="header-heading-item">
+          <Button variant="primary" title="New blog" size="md" />
+        </section>
+      </section>
+      <section className="header-filter">
+        <nav className="header-navigation">
+          <Button tag="a" title="All topic" path="/" />
```
Comment (huypham-agilityio) [inline practices/practice-one/src/components/Header/Header.tsx]: should create data for these and map them

Comment (DuongPham-dev) [reply practices/practice-one/src/components/Header/Header.tsx]: I'll do it now

File: practices/practice-one/src/components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+import cinndy from "assets/images/Cinndy.jpg";
+import cector from "assets/icons/Vector.svg";
+import Heading from "components/Heading/Heading";
+import Button from "components/Button/Button";
+import Paragraph from "components/Paragraph/Paragraph";
+import Input from "components/Input/Input";
+
+import "./header.css";
+
+function Header() {
```
Comment (huypham-agilityio) [inline practices/practice-one/src/components/Header/Header.tsx:12]: dont put it in component, move to constants instead

Comment (DuongPham-dev) [reply practices/practice-one/src/components/Header/Header.tsx:12]: yes sir

**Review by huypham-agilityio — APPROVED** [review]

**Pull request merged**